### PR TITLE
Item Groups For Random Encounters

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
None

#### Purpose of change
I had done these and then forgot. Some reasonable item groups for characters to pull from when you need someone wearing not dumb shoes and not dumb pants in the apocalypse. Could be further customized per faction but I think this is a good start.

#### Describe the solution
Added a number of "Practical" groups for any layer you might want to fill.

It looks like some groups for certain classes already don't work great or at all... gonna investigate and add some more stuff to this PR.

### CODE REVIEW

File: data/json/itemgroups/Clothing_Gear/practical_survivor_items.json [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    "subtype": "distribution",
+    "//": "Assortment of practical underwear characters would wear during the apocalpyse.",
+    "items": [
+      { "item": "boxer_briefs" },
+      { "item": "boxer_shorts" },
+      { "item": "leggings" },
+      { "item": "boy_shorts" },
+      { "item": "briefs" },
+      { "item": "panties" },
+      { "item": "sports_bra" },
```
Comment (anothersimulacrum) [inline data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:48]: One of these is not like the others.

Comment (Termineitor244) [reply data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:48]: Which could it be?...

Comment (anothersimulacrum) [reply data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:48]: All of these cover the legs, this one covers the chest.

Comment (Termineitor244) [reply data/json/itemgroups/Clothing_Gear/practical_survivor_items.json]: I know haha, it was a joke (or sarcasm?) But I guess written communication is harder

File: data/json/itemgroups/Clothing_Gear/practical_survivor_items.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      { "item": "panties" },
+      { "item": "under_armor_shorts" }
+    ]
+  },
+  {
+    "id": "practical_socks",
+    "type": "item_group",
+    "//": "assortment of practical socks characters would wear during the apocalypse",
+    "subtype": "distribution",
+    "items": [ { "item": "socks" }, { "item": "socks_wool" } ]
```
Comment (Termineitor244) [inline data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:56]:
```suggestion
    "items": [ { "item": "socks" }, { "item": "socks_wool" }, { "item": "socks_ankle" } ]
```

File: data/json/itemgroups/Clothing_Gear/practical_survivor_items.json [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    "subtype": "distribution",
+    "//": "Assortment of practical underwear characters would wear during the apocalpyse.",
+    "items": [
+      { "item": "boxer_briefs" },
+      { "item": "boxer_shorts" },
+      { "item": "leggings" },
+      { "item": "boy_shorts" },
+      { "item": "briefs" },
+      { "item": "panties" },
+      { "item": "under_armor_shorts" }
```
Comment (Termineitor244) [inline data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:48]:
```suggestion
      { "item": "under_armor_shorts" },
      { "item": "shorts" }
```

File: data/json/itemgroups/Clothing_Gear/practical_survivor_items.json [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+    "type": "item_group",
+    "subtype": "distribution",
+    "//": "Assortment of practical underwear characters would wear during the apocalpyse.",
+    "items": [
+      { "item": "boxer_briefs" },
+      { "item": "boxer_shorts" },
+      { "item": "leggings" },
+      { "item": "boy_shorts" },
+      { "item": "briefs" },
+      { "item": "panties" },
```
Comment (Termineitor244) [inline data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:47]: You may want to remove this unless you are dividing these groups by gender (same reason why I'm not suggesting the bra and sports bra to the torso undergarments).
```suggestion
```

File: data/json/itemgroups/Clothing_Gear/practical_survivor_items.json [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+    "type": "item_group",
+    "subtype": "distribution",
+    "//": "Assortment of practical under shirts and underwear characters would wear during the apocalpyse.",
+    "items": [
+      { "item": "army_top" },
+      { "item": "tank_top" },
+      { "item": "under_armor" },
+      { "item": "tshirt" },
+      { "item": "tshirt_tour" },
+      { "item": "longshirt" }
```
Comment (Termineitor244) [inline data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:20]:
```suggestion
      { "item": "longshirt" },
      { "item": "thermal_shirt" }
```

File: data/json/itemgroups/Clothing_Gear/practical_survivor_items.json (hunk 4 above)
Comment (Termineitor244) [inline data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:47]: I suggest removing this unless you are planning to divide these groups by gender (same reason why I didn't suggest adding the bra and sports bra to the torso undergarments).
```suggestion
```

File: data/json/itemgroups/Clothing_Gear/practical_survivor_items.json (hunk 3 above)
Comment (Termineitor244) [inline data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:48]:
```suggestion
      { "item": "under_armor_shorts" },
      { "item": "shorts" }
```

File: data/json/itemgroups/Clothing_Gear/practical_survivor_items.json (hunk 2 above)
Comment (Termineitor244) [inline data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:56]:
```suggestion
    "items": [ { "item": "socks" }, { "item": "socks_wool" } , { "item": "ankle_socks" } ]
```

**Review by Termineitor244 — COMMENTED** [review]
Some suggestions for more practical items! And I suggest dividing some of these groups by gender or just putting unisex clothing.

File: data/json/itemgroups/Clothing_Gear/practical_survivor_items.json (hunk 3 above)
Comment (bombasticSlacks) [reply data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:48]: shorts are normal layer IIRC

Comment (Termineitor244) [reply data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:48]: Oh, sorry, I'm doing this on mobile so I remembered wrong.

Comment (bombasticSlacks) [reply data/json/itemgroups/Clothing_Gear/practical_survivor_items.json:48]: all good I appreciate the suggestions :)

**Comment by Night-Pryanik** [comment]
These new item groups should be used somewhere.

**Comment by bombasticSlacks** [comment]
Erk wanted them for one of his PRs. Otherwise I can write an encounter that uses them.

**Comment by Night-Pryanik** [comment]
> Otherwise I can write an encounter that uses them.

Yes, please.

**Comment by bombasticSlacks** [comment]
Not realistically going to be completed

**Pull request closed without merging**